.github/workflows/rust.yml

```diff
@@ -20,7 +20,6 @@ jobs:
     - name: Build
       run: cargo build --verbose
     # run all tests
-    - uses: actions/checkout@v3
     - name: Run tests
       run: cargo build && cargo test --verbose
```

Comment:
How about remove the `build` since it is already executed in `cross-testing`?

Reply:
Yeah, I think build is redundant here.